src/modules/ims_ipsec_pcscf/cmd.c

```diff
@@ -562,6 +560,24 @@ int ipsec_forward(struct sip_msg* m, udomain_t* d)
     struct pcontact_info ci;
     pcontact_t* pcontact = NULL;
     int ret = IPSEC_CMD_FAIL; // FAIL by default
+    unsigned char dst_proto = PROTO_UDP;
```

Comment:
Is this hardcoded to PROTO_UDP on purpose?

Reply:
It's not an error, just default value.